style.css

```
@@ -0,0 +1,82 @@
+body{
+    background-color: rgb(249, 246, 243);
+}
+
+header{
+    position: fixed;
+    text-align: center;
+    left: 0;
+    right: 0;
+    top:0;
+    width: auto;
+    height: 5vh;
+    display: flex;
+    color:rgb(83, 81, 81);
+    justify-content: space-around;
+}
+.top{
+    position: sticky;
+ top: 30px;
+}
```

The styles should start from the same vertical line, two spaces from the begining.
```suggestion
.top {
    position: sticky;
    top: 30px;
}
```